### DISCUSSION TITLE
Modelling choices for diphtheria

### DISCUSSION THREAD

**Discussion by pratikunterwegs** [epiverse-trace/epiverse-trace.github.io#136] [General]
This discussion is about modelling choices for a diphtheria model to be added to {epidemics}.

The model will be based on [Finger et al. (2019)](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6413455/) for IDP/refugee camps.

Could I please hear preferences on some modelling choices. Where there are clear options I have marked the **parsimonius, less time-intensive option**.

1. The model assumes homogeneous social contacts when calculating force of infection. Options:
 a. Implement heterogeneous social contacts,
 b. **Assume camps have small spatial scale that results in homogeneous social contacts.**

2. The model assumes some proportion of the 5 - 14 age group to be vaccinated and immune. Three options:
 a. Allow one/multiple vaccination compartments and add vaccination dynamics (taken from [this WHO position paper](https://www.who.int/teams/immunization-vaccines-and-biologicals/policies/position-papers/diphtheria)),
 b. **Allow model parameter for p_vaccinated and remove this proportion from susceptibles.**
 c. **No model parameter for vaccinations** and users should pass correct number of susceptibles

3. The model is an age-structured meta-population model with a flexible number of locations. Three options:
 a. Include location structure at this stage,
 b. **Implement a single location model, adding location structure later,**
 c. **Implement only a single location model.**

4. The model assumes no movement or contacts between camps, and lists this as a drawback in the Discussion. Please comment on whether this should be considered/added as a transition matrix, or in the form of events modifying population sizes in each camp.

Tagging @rozeggo @BlackEdder @adamkucharski and @sbfnk for comments.

**Comment by rozeggo** [comment]
Hi Pratik,

Great. 
1b sounds good. We dont know mixing patterns or age structure of camps very well. 
2b also ok. I dont see 3 options in 2 though! :)
3c I think is the right choice for now. I have contacted colleagues at MSF about movements.
4. You and I have discussed before about whether we include change in camp size, e.g. via inflow/outflow of population, and I am waiting to hear back from colleagues at MSF for their thoughts.

This seems good for now.

**Reply by pratikunterwegs** [reply]
Thanks - I'll go with 1b, 2b, and 3c for now. Regarding 2, option c would be to ignore vaccination structure and let users pass the correct number of susceptibles. I'll add that to the list.

Regarding 4, I've modified the original post to clarify that this refers to connections between camps that do not change their overall size but could spread infection (a sort of inter-camp contact matrix). Since we are going with 3c for now, I'll leave this as a future option. I will update this option once we know more from MSF - it would be great if they could also feed back here directly for transparency.

**Reply by sbfnk** [reply]
Thanks @pratikunterwegs for tagging but I don't thin I'm much use here. Any model choice would depend on the model aims which I'm not aware of.

**Reply by pratikunterwegs** [reply]
Thanks - you and Adam are authors on the paper so I thought you might have some feedback.

**Reply by sbfnk** [reply]
What I meant is: if the aim is to reproduce the analysis then I'd suggest to use the same model. If not then the answer to your question depends on what the model is to be used for.
